tailwind.config.js

```diff
@@ -9,19 +9,27 @@ module.exports = {
         light: '#25ADE3'
       },
       grey: {
-        base: '#1F2122',
-        secondary: '#F2F2F1',
-        100: '#222425',
-        200: '#27292A',
-        300: '#313334',
-        400: '#3B3D3E',
-        500: '#585A5A'
+        base: '#121212',
+        0: 'rgba(255,255,255,0)',
```

Comment:
This feels a bit odd defining transparent colours, because if we use them as say overlay colours in the future or something, we would end up with a transparent effect. Might it be better to leave the grey definitions as is, but instead extend the bg-opacity modifiers as required?

Reply:
good call, removed